portfolio/src/main/webapp/css/about.css

```diff
@@ -0,0 +1,59 @@
+.blackBackgroundImage, .longGroveImage, .stanfordImage, .hobbiesImage {
+  position: relative;
+  opacity: 1;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
```

Comment:
Can we pull this out into a common `.parallax-background-image`, and then we can use it on a div like `class="parallax-background-image long-grove-background"`?

Reply:
sounds good